src/components/VHeader/VHeader.vue

```diff
@@ -204,6 +204,7 @@ const VHeader = defineComponent({
       })
       router.push(newPath)
       await mediaStore.fetchMedia()
+      document.activeElement?.blur()
```

Comment:
Can we move the blurring before the fetching media? Otherwise when you search there is a delay while the new results are fetched before the keyboard disappears.